articles/200_migration_guide_from_ros1.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: default
+title: Migration guide from ROS 1
+permalink: articles/migration_guide_from_ros1.html
+abstract:
+  This article describes the steps to migrate a ROS 1 package to ROS 2.
+published: true
+author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+## Prerequisite
+
+Before being able to migrate a ROS 1 package to ROS 2 all of its dependencies must be available in ROS 2.
+
+
+## Migration steps
+
+### Package naming
+
+Packages containing messages and / or services which should be supported by the [ROS bridge](https://github.com/ros2/ros1_bridge/) need to have a different name in ROS 1 and ROS 2.
+Therefore the suffixes `_msgs` and `_srvs` are replaced (and merged in a single package) with the suffix `_interfaces`.
+
+
+### Package manifests
+
+ROS 2 only support the format 2 of the package specification which is defined in [REP 140](http://www.ros.org/reps/rep-0140.html).
+Therefore the `package.xml` file must be update to format 2 if it uses format 1.
+Since ROS 1 support both formats (1 as well as 2) it is save to perform that conversion in the ROS 1 package.
+
+Some packages might have different names in ROS 2 so the dependencies might need to be updated accordingly.
+
+
+### Messages and services
+
+Message files ending in `.msg` must be located in the subfolder `msg`.
```

Comment:
This line and the one below need a reword, as there's some room for confusion. I interpret as either:
* Message (service) files **must end** in `.msg` (`.srv`) **and** must be  located in...
* Does the convention allow message and service files to have other extensions than `.msg` and `.srv`?. If so, should they also live in the `msg` and `srv` folders, or elsewhere?.